common/evmcompat/helpers.go

```diff
@@ -94,7 +95,7 @@ func RecoverAddressFromTypedSig(hash []byte, sig []byte) (common.Address, error)
 	case SignatureType_EOS:
 		return recoverAddressFromEos(hash, sig)
 	case SignatureType_EOS_SCATTER:
-		return recoverAddressFromEosScatter(hash, sig)
+		return recoverAddressFromEosScatter(hash, sig[1:])
```

Comment:
-> return recoverAddressFromEosScatter(hash, `sig`) Might as well be consistant.